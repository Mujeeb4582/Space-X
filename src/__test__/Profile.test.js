// file for profile test
import TestRenderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../redux/configureStore';
import Profile from '../routes/profile';

describe('Test Profile component', () => {
  it('should renders Profile component', () => {
    const ProfileComponent = TestRenderer.create(
      <Provider store={store}><Router><Profile /></Router></Provider>,
    ).toJSON();
    expect(ProfileComponent).toMatchSnapshot();
  });
});
