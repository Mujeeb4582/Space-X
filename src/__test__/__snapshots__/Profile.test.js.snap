// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Profile component should renders Profile component 1`] = `
<main
  className="mt-6 px-12 w-full flex flex-col gap-6 md:flex-row"
>
  <section
    className="md:w-1/2"
  >
    <h2
      className="text-2xl font-semibold pl-2 pb-4"
    >
      My Rockets
    </h2>
    <ul
      className="flex flex-col border-t-2 border-gray-500"
    >
      <span
        className="flex h-24 text-lg border-2 border-t-0 border-gray-500 items-center p-4"
      >
        You have no reserved rockets
      </span>
    </ul>
  </section>
  <section
    className="w-1/2"
  >
    <h2
      className="text-2xl font-semibold pl-2 pb-4"
    >
      My Missions
    </h2>
    <ul
      className="flex flex-col"
    />
  </section>
</main>
`;
